test/src/test/resources/logback-test.xml

```diff
@@ -61,7 +61,7 @@
     <!-- Control logging levels for individual components here. -->
     <logger name="org.corfudb.runtime.object" level="INFO"/>
     <logger name="org.corfudb.runtime.clients" level="INFO"/>
-    <logger name="org.corfudb.infrastructure" level="INFO"/>
+    <logger name="org.corfudb.infrastructure" level="WARN"/>
```

Comment:
why did we change the test logging to WARN?

Reply:
Sorry for the confusion, those from the second commit are just temporary for debugging the flaky test.